opcpa_tpr_config/new_neh_bay2_config.yaml

```diff
@@ -0,0 +1,125 @@
+# Configuration file for NEH OPCPA lasers
+
+main:
+    xpm_pv: "DAQ:NEH:XPM:0"
+    meta_pv: "TPG:SYS0:1:DST04"
+    engine1: "4"
+    engine2: "5"
+    title: "NEH Laser Hall Bay 2 OPCPA Rep. Rate Configuration"
+    bay: "Bay 2"
+
+    laser_database: "neh_bay2_db.json"
+
+    # TPR Channels
+    devices:
+      Carbide_Goose:
+        rbvs: ["name", "eventrate", "ratemode", "seqcode", "width_setpoint", "delay_setpoint", "operation", "enable_ch_cmd"]
```

Comment:
Yeah, you're probably right. I can make an issue for that. But this is how the old code worked too lol.

Reply:
I don't think it's a major change or even in scope for this PR. Just a nitpick that might be worth touching later — unless there's some reason this can't work (and there definitely might be).